omnibus/resources/chef-workstation/pkg/entitlements.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
+    <true/>
+  </dict>
+</plist>
```

Comment:
Not sure if this file needs a new line

Reply:
I don't think it's required, but from a style perspective maybe we should. I'm going to merge without and we'll figure it out.